### DISCUSSION TITLE
Host or Bridge Network?

### DISCUSSION THREAD

**Discussion by jgrizzle33** [qdm12/gluetun#571] [Ideas]
Is it normal behavior to start the docker, and have all container automatically connect under the VPN?

Under the Gluetun docker, I do have the network set to host - not sure if I did that or not.  Don't get me wrong, all dockers need to be running under Gluetun so if this work I'm happy, but I just found it a little odd.

Should it be Host or Bridge?

**Comment by qdm12** [comment]
Don't run it on the host network ideally, it will probably mess up your firewall if you shut the container down. The assumption here is that gluetun's firewall is isolated so it's fine to set its own policies and block everything but the vpn, without restoring the state of iptables. Although feel free to create an issue for that, I could have a look.

You should instead use one of these: https://github.com/qdm12/gluetun/wiki/Connect-to-gluetun#external-container-to-gluetun
